src/core_functions/scalar/list/array_slice.cpp

```diff
@@ -180,13 +361,18 @@ static unique_ptr<FunctionData> ArraySliceBind(ClientContext &context, ScalarFun
 	return make_uniq<VariableReturnBindData>(bound_function.return_type);
 }
 
-ScalarFunction ListSliceFun::GetFunction() {
+ScalarFunctionSet ListSliceFun::GetFunctions() {
 	// the arguments and return types are actually set in the binder function
 	ScalarFunction fun({LogicalType::ANY, LogicalType::BIGINT, LogicalType::BIGINT}, LogicalType::ANY,
 	                   ArraySliceFunction, ArraySliceBind);
-	fun.varargs = LogicalType::ANY;
+	//		fun.varargs = LogicalType::ANY; // Do we need this?
```

Comment:
I don't think that we need this. We cover `LIST` and `VARCHAR` input with `LogicalType::ANY`. Then, we add the no-step-function. Then, we add the step-function. So should be fine.